build.gradle

```diff
@@ -469,6 +484,7 @@ project('test:integration') {
         compile project(':test:testcommon')
         compile group: 'junit', name:'junit', version: junitVersion
         compile group: 'org.apache.curator', name: 'curator-test', version: apacheCuratorVersion
+        compile project(path: ':shared:security', configuration: 'testOutput')
```

Comment:
Why is this needed, and what is it doing?

Reply:
This line is needed as the project integration has compile dependency over the test present in the project security so when this is run gradle will run the jarTest task in the security project in the build.gradle.